src/modules/subscribers/subscribers.test.ts

```diff
@@ -74,6 +74,35 @@ describe("Subscribers", () => {
         }
     });
 
+    it("Update a subscriber", async () => {
+        const randomInt = getRandomInt();
+
+        const params: CreateOrUpdateSubscriberParams = {
```

Comment:
`UpdateSubscriberParams` has to be included and used here
```suggestion
        const params: UpdateSubscriberParams = {
```

Reply:
@Aky22 also please import `UpdateSubscriberParams` `from "./campaigns.types"`

Reply:
I assume you mean from `./subscribers.types` . It's done :slightly_smiling_face: